### PR TITLE
fix: edits from Andrew

### PR DESCRIPTION
**Pull request by johnnyreilly**

### CODE REVIEW

File: blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md [hunk 1]
```diff
@@ -68,7 +68,9 @@ src/telemetry.ts:9:1 - error TS1287: A top-level 'export' modifier cannot be use
 export async function sendTelemetry({
 ```
 
-It turns out that `verbatimModuleSyntax` doesn't work with CommonJS - and my codebase is CommonJS.
+The big thing with `verbatimModuleSyntax` is you can't write ESM syntax in CJS files.
+
+In TypeScript 5.0, a new compiler option called `verbatimModuleSyntax` was introduced to help TypeScript authors know exactly how their `import` and `export` statements will be emitted. When enabled, the flag requires imports and exports in input files to be written in the form that will undergo the least amount of transformation before emit. So if a file will be emitted as ESM, imports and exports must be written in ESM syntax; if a file will be emitted as CJS, it must be written in the CommonJS-inspired TypeScript syntax (`import fs = require("fs")` and `export = {}`). This setting is particularly recommended for Node projects that use mostly ESM, but have a select few CJS files. It is not recommended for projects that currently target CJS, but may want to target ESM in the future.
```
Comment (andrewbranch) [inline blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md:73]: What do you think about something like this (GH isn’t letting me use the suggestion UI for some reason):

---

It turns out that in `verbatimModuleSyntax`, you can't write ESM syntax in files that will emit as CommonJS - which is exactly what my codebase is doing. [Andrew Branch](https://github.com/andrewbranch), who is part of the TypeScript team, sent me an explanation from a draft of some [new TypeScript docs](https://github.com/microsoft/TypeScript/issues/52593):

> In TypeScript 5.0, a new compiler option called `verbatimModuleSyntax` was introduced to help TypeScript authors know exactly how their `import` and `export` statements will be emitted. When enabled, the flag requires imports and exports in input files to be written in the form that will undergo the least amount of transformation before emit. So if a file will be emitted as ESM, imports and exports must be written in ESM syntax; if a file will be emitted as CJS, it must be written in the CommonJS-inspired TypeScript syntax (`import fs = require("fs")` and `export = {}`). This setting is particularly recommended for Node projects that use mostly ESM, but have a select few CJS files. It is not recommended for projects that currently target CJS, but may want to target ESM in the future.

**Review by andrewbranch — COMMENTED** [review]
❤️

File: blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md (hunk 1 above)
Comment (johnnyreilly) [reply blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md:73]: Looks good - I think I follow what you mean. Just popped out for a quick walk before the next thunderstorm kicks in. Will apply your suggestion when back!

Comment (johnnyreilly) [reply blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md]: Just applied the change (I think) - this what you had in mind?

File: blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md [hunk 2]
```diff
@@ -68,11 +68,11 @@ src/telemetry.ts:9:1 - error TS1287: A top-level 'export' modifier cannot be use
...
+> In TypeScript 5.0, a new compiler option called `verbatimModuleSyntax` was introduced to help TypeScript authors know exactly how their `import` and `export` statements will be emitted. When enabled, the flag requires imports and exports in input files to be written in the form that will undergo the least amount of transformation before emit. So if a file will be emitted as ESM, imports and exports must be written in ESM syntax; if a file will be emitted as CJS, it must be written in the CommonJS-inspired TypeScript syntax (`import fs = require("fs"`) and `export = {}`). This setting is particularly recommended for Node projects that use mostly ESM, but have a select few CJS files. It is not recommended for projects that currently target CJS, but may want to target ESM in the future.
```
Comment (andrewbranch) [inline blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md:73]: There’s a transposed paren and backtick here

```suggestion
> In TypeScript 5.0, a new compiler option called `verbatimModuleSyntax` was introduced to help TypeScript authors know exactly how their `import` and `export` statements will be emitted. When enabled, the flag requires imports and exports in input files to be written in the form that will undergo the least amount of transformation before emit. So if a file will be emitted as ESM, imports and exports must be written in ESM syntax; if a file will be emitted as CJS, it must be written in the CommonJS-inspired TypeScript syntax (`import fs = require("fs")` and `export = {}`). This setting is particularly recommended for Node projects that use mostly ESM, but have a select few CJS files. It is not recommended for projects that currently target CJS, but may want to target ESM in the future.
```

File: blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md [hunk 3]
```diff
@@ -68,9 +68,11 @@ src/telemetry.ts:9:1 - error TS1287: A top-level 'export' modifier cannot be use
 export async function sendTelemetry({
 ```
 
-It turns out that `verbatimModuleSyntax` doesn't work with CommonJS - and my codebase is CommonJS.
+It turns out that in verbatimModuleSyntax, you can't write ESM syntax in files that will emit as CommonJS - which is exactly what my codebase is doing. [Andrew Branch](https://github.com/andrewbranch), who is part of the TypeScript team, sent me an explanation from a draft of some new TypeScript docs:
```
Comment (andrewbranch) [inline blog-website/blog/2023-05-09-typescript-5-importsnotusedasvalues-error-eslint-consistent-type-imports/index.md:71]:
```suggestion
It turns out that in `verbatimModuleSyntax`, you can't write ESM syntax in files that will emit as CommonJS - which is exactly what my codebase is doing. [Andrew Branch](https://github.com/andrewbranch), who is part of the TypeScript team, sent me an explanation from a draft of some new TypeScript docs:
```

**Pull request merged**